src/media/event.rs

```diff
@@ -122,7 +122,7 @@ pub struct MediaData {
 
     /// RTP header extensions for this media data. This is taken from the
     /// first RTP header.
-    pub ext_vals: ExtensionValues,
```

Comment:
This is "only" 184 bytes large and boxing it adds another allocation to our hottest path(parsing incoming RTP packets). Is it worth it?

Reply:
Agree. Not worth it. I remove that.